modules/claims_api/config/schemas/526.json

```diff
@@ -1322,10 +1322,10 @@
       "example": false
     },
     "claimDate": {
```

Comment:
JSON schema has a `format` field with a value of `date-time` that we should consider using here.
[REF](https://json-schema.org/understanding-json-schema/reference/string.html#dates-and-times)

Reply:
another option could be the [`oneOf` property](https://json-schema.org/understanding-json-schema/reference/combining.html#oneof), i.e., something like this:
```json
{
  "oneOf": [
    { "type": "date" },
    { "type": "date-time" }
  ]
}
```
instead of a string. that has the added benefit of having the regex baked in